WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -264,8 +265,8 @@ - (RemoteReaderPost *)formatPostDictionary:(NSDictionary *)dict
     post.blogName = [self siteNameFromPostDictionary:dict];
     post.blogDescription = [self siteDescriptionFromPostDictionary:dict];
     post.blogURL = [self siteURLFromPostDictionary:dict];
-    post.commentCount = [dict numberForKey:@"comment_count"];
-    post.commentsOpen = [[dict numberForKey:@"comments_open"] boolValue];
+    post.commentCount = [discussionDict numberForKey:@"comment_count"];
+    post.commentsOpen = [[discussionDict numberForKey:@"comments_open"] boolValue];
```

Comment:
This works for the sites endpoints but the read endpoints return the information the old way and don't have a `discussion` field. 
Maybe add something like `NSDictionary *commentsDict = [dict objectForKey:@"discussion"] ? [dict dictionaryForKey:@"discussion"] : dict;` to normalize retrieval?